flipper-plugin-realm/src/components/DataTable.tsx

```diff
@@ -66,24 +72,24 @@ export const schemaObjToColumns = (schema: SchemaObject): ColumnType[] => {
   });
 };
 
-export const DataTable = ({
-  columns,
-  objects,
-  schemas,
-  currentSchema,
-  generateMenuItems,
-  setdropdownProp,
-  dropdownProp,
-  scrollX,
-  scrollY,
-  setNewInspectionData,
-  enableSort,
-  hasMore,
-  totalObjects,
-  fetchMore,
-  clickAction,
-}: // rowSelection
-PropertyType) => {
+export const DataTable = (dataTableProps: DataTableProps) => {
+  const {
+    columns,
+    objects,
+    schemas,
+    currentSchema,
+    generateMenuItems,
+    setdropdownProp,
+    dropdownProp,
+    scrollX,
+    scrollY,
+    setNewInspectionData,
+    enableSort,
+    hasMore,
+    totalObjects = 0,
+    fetchMore = () => undefined, 
+    clickAction,
+  } = dataTableProps;
```

Comment:
Any reason to destructure the properties here as opposed to in the function declaration?

Reply:
It is passed to `NestedTable` properties

Reply:
Ah, yeah that's fair